ee/desktop/release-notes.md

```diff
@@ -15,6 +15,13 @@ For information on system requirements, installation, and download, see:
 
 For Docker Enterprise Engine release notes, see [Docker Engine release notes](/engine/release-notes).
 
+## Version 2.1.0.5
+2019-09-16
+
+### Bug fixes and minor changes
+
+ Mac: All binaries included in Docker Desktop are now notarized so that they can run on macOS Catalina.  For more information, see [Notarization Requirement for Mac Software](https://developer.apple.com/news/?id=06032019i).
```

Comment:
nit: there is a double space before `For more information`

Reply:
Aah. Good spot. I'll fix this.

Reply:
Fixed in commit `dddf555`